generated/nidigitalpattern/nidigitalpattern_service.cpp

```diff
@@ -1478,9 +1474,11 @@ namespace nidigitalpattern_grpc {
         if (status == 0) {
           CopyBytesToEnums(expected_pin_states, response->mutable_expected_pin_states());
           response->set_expected_pin_states_raw(expected_pin_states);
+          response->mutable_expected_pin_states()->Resize(actual_num_pin_data, 0);
```

Comment:
I accidentally exposed a missing case here. Arrays of enum-byte-arrays were not going through the ivi-dance resizing. That got fixed by rejiggering that codepath.